hbook/ntup/ntup_example_test.go

```diff
@@ -47,3 +48,67 @@ func ExampleNtuple_scanH2D() {
 	// XStdErr:    0.957427
 	// YStdErr:    0.957427
 }
+
+func ExampleNtuple_scanH() {
```

Comment:
please drop the final `H` (to make clearer this is an example about the `Scan` method of `Ntuple`.)